PRIVACY.md

```diff
@@ -34,10 +34,11 @@ The Name Suggestion Index is a feature in iD to aid users in consistently and ac
 
 ### Contact
 
-If you believe that we hold information that would allow us to correct, amend, or delete inaccurate information about you or if you believe that information about you has been processed in violation of this privacy policy or applicable law, please email us at privacy@ideditor.org.
+If you believe that we hold information that would allow us to correct, amend, or delete inaccurate information about you or if you believe that information about you has been processed in violation of this privacy policy or applicable law, please email us at [legal-questions@osmfoundation.org](mailto:legal-questions@osmfoundation.org)</a>.
 
 ### Changelog
 
+* _2021-Dec-xx_ - Remove false statement which claimed that _the OpenStreetMapFoundatino does not operate iD_.
```

Comment:
Not every instance of iD is run from OSMF servers. Does the privacy policy need to account for independent deployments, or is it up to them to provide their own privacy policies even if nothing is being customized?

Reply:
What we probably should have done in #7040 was put the privacy policy and its datestamp in the `dist/` folder in a .json file with the other iD assets, then have the display code look for that file and display its contents only if it exists and is newer than what has been displayed previously.  This would let other sites override it easily, or omit it entirely. 
 The iD embedded in other sites like openstreetmap.org or openhistoricalmap.org are covered by their respective privacy policies.

A lot of the privacy policy discussion leading up to and on #7040 didn't make any sense for iD (a client side app that collects no data, stores no data, and sets no cookies), but we tried.  (more info on privacy notices can be found [here](https://gdpr.eu/privacy-notice/))